pyproject.toml

```diff
@@ -12,6 +12,12 @@ dependencies = [
 [project.scripts]
 promdens = "promdens:main"
 
+[project.optional-dependencies]
+tests = [
```

Comment:
These optional dependencies (also called "extras") are only meant for developers, not users. They can be installed with 

```
pip install -e .[tests]
```